gradle.properties

```diff
@@ -4,6 +4,15 @@ pluginName_ = Amazon Ion
 pluginDescriptionFile = plugin-description.html
 pluginChangeNotesFile = plugin-changenotes.html
 
+# If JetBrains makes a breaking change, bump this to the latest RELEASE version and fix/update the plugin.
+supportedSinceIdeVersion = 231.1
+# We could leave this unset so that it will automatically support newer IDE versions until JetBrains
+# marketplace determines that there's an incompatibility with a newer version. However, if we do that,
+# then the `printProductReleases` won't display a full range of products; it will only display the
+# "supportedSinceIdeVersion" product. So, instead, we set this so far in the future that IntelliJ
+# might not even exist by then.
+supportedUntilIdeVersion = 993.*
```

Comment:
Is it worth linking to one of these, here? Also might as well use `999`, who's to say that by 2099 they won't have 9+ releases per year because we're in the middle of an IDE singularity or something? :P

- https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#build-number-format
- https://blog.jetbrains.com/blog/2016/03/09/jetbrains-toolbox-release-and-versioning-changes/

Reply:
Look, if there's an IDE singularity in 2099, that's someone else's problem. I'm going to be retiring in 2098 at the latest. 😉

Reply:
I'd consider it an unqualified success (on the part of Ion and IntelliJ both) if anyone ever sees a problem from this.